api/include/opentelemetry/nostd/internal/absl/base/options.h

```diff
@@ -69,8 +69,31 @@
 
 // Include a standard library header to allow configuration based on the
 // standard library in use.
-#ifdef __cplusplus
-#include <ciso646>
+// Using C++20 feature-test macros when possible, otherwise fall back to
+// ciso646/iso646.h.There are warnings when includeing ciso646 in C++17 mode
```

Comment:
```suggestion
// ciso646/iso646.h. There are warnings when including ciso646 in C++17 mode
```

Reply:
Is it possible to add a link to the issue of compiling ciso646 in C++17 mode?

Reply:
Thanks, done.